### PR TITLE
Ultra: Added exposure time for pointing frame

### LINKED ISSUE #970: Ultra: Add Exposure Time for Pointing Frame
### Summary

_No response_

### Requirements

The more specific the better.
  - [ ]
  - [ ]

### Related tickets

_No response_

### PR DESCRIPTION
**Pull request by laspsandoval**
# Change Summary

## Overview
Ultra has a constant exposure time file that is binned by az and el.

## New Files - lookup tables
- One lut for exposure time. 

## Updated Files
- ultra_l1c_pset_bins.py
   - Added exposure time to pointing frame (DPS)

## Testing
- test_ultra_l1c_pset_bins.py
   - Tests ultra_l1c_pset_bins.py

### CODE REVIEW

File: imap_processing/ultra/l1c/ultra_l1c_pset_bins.py [hunk 1]
```diff
@@ -145,3 +148,29 @@ def get_histogram(
...
+        Path to file containing constant exposure data.
+    n_spins : int
+        Number of spins per pointing.
+    sensor : str
+        Sensor (45 or 90).
+
+    Returns
+    -------
+    exposure : np.ndarray
+        A 2D array with dimensions (az, el).
```
Comment (tmplummer) [inline imap_processing/ultra/l1c/ultra_l1c_pset_bins.py:171]: Is it assumed that the shape of this 2D array will always match the pset bins in azimuth and elevation? I could see it being beneficial to read in the data and interpolate to whatever `(az, el)` grid is passed in but that may never be necessary.

File: imap_processing/ultra/lookup_tables/dps_grid45_compressed.cdf
Comment (tmplummer) [inline imap_processing/ultra/lookup_tables/dps_grid45_compressed.cdf:1]: Based on the size of this file, I assume it was saved with compression turned on?

**Review by tmplummer — APPROVED** [review]
Just have the two quick questions. Implementation looks good.

File: imap_processing/ultra/lookup_tables/dps_grid45_compressed.cdf
Comment (laspsandoval) [reply imap_processing/ultra/lookup_tables/dps_grid45_compressed.cdf:1]: Correct.

File: imap_processing/ultra/l1c/ultra_l1c_pset_bins.py (hunk 1 above)
Comment (laspsandoval) [reply imap_processing/ultra/l1c/ultra_l1c_pset_bins.py:171]: It will always match and therefore interpolation will not be required. Good idea though!

**Pull request merged**